api/app/models/bookings/appointments.py

```diff
@@ -21,13 +21,17 @@ class Appointment(Base):
 
     appointment_id = db.Column(db.Integer, primary_key=True, autoincrement=True, nullable=False)
     office_id = db.Column(db.Integer, db.ForeignKey("office.office_id"), nullable=False)
+    service_id = db.Column(db.Integer, db.ForeignKey("service.service_id"), nullable=False)
     start_time = db.Column(UtcDateTime, nullable=False)
     end_time = db.Column(UtcDateTime, nullable=False)
     category = db.Column(db.String(255), nullable=False)
```

Comment:
What is this field used for?

Reply:
<img width="1016" alt="Screen Shot 2019-03-29 at 9 44 05 AM" src="https://user-images.githubusercontent.com/29988052/55248538-380aee80-5207-11e9-8bd6-1ed0dc11b981.png">

Reply:
Going off a slack conversation on this one.

Reply:
Yea, but why is category here, and a string?